docs/index.bs

```diff
@@ -1590,25 +1588,33 @@ spec: webappsec-referrer-policy; urlPrefix: https://w3c.github.io/webappsec-refe
                 1. Let |done| be false.
                 1. Let |potentialResponse| be a copy of |response|'s associated [=Response/response=], except for its [=response/body=].
                 1. If |response|'s [=response/body=] is non-null, run these substeps:
-                    1. Let |reader| be the result of [=get a reader|getting a reader=] from |response|'s [=response/body=]'s [=stream=].
+                    1. Let |reader| be the result of [=ReadableStream/getting a reader=] from |response|'s [=response/body=]'s [=stream=].
+                    1. Let |pullAlgorithm| be an action that runs these subsubsteps:
+                        1. Let |readRequest| be a new [=read request=] with the following [=struct/items=]:
+                            : [=read request/chunk steps=], given |chunk|
+                            ::
+                                1. Assert: |chunk| is a {{Uint8Array}}.
+                                1. Append the bytes represented by |chunk| to |bytes|.
+                                1. Perform ! [=DetachArrayBuffer=](|chunk|.\[[ViewedArrayBuffer]]).
+                            : [=read request/close steps=]
+                            ::
+                                1. Set |end-of-body| to true.
+                            : [=read request/error steps=]
+                                1. [=ReadableStream/error=] |newStream| with a {{TypeError}}.
```

Comment:
The markup seems inconsistent here. With the error steps, the steps aren't in a definition item. Given that they're both a single step, do they need to be in an ordered list at all? Should it be:

```suggestion
                            : [=read request/close steps=]
                            :: Set |end-of-body| to true.
                            : [=read request/error steps=]
                            :: [=ReadableStream/error=] |newStream| with a {{TypeError}}.
```

Also, is it typical to use lower case for "close steps" and "error" even though they're at the beginning of a sentence?

Reply:
Thanks for catching the missing `::`!

I find it clearer when steps are numbered, but we could remove it if you'd prefer; the reader can understand.

I don't think of `<dd>`s as "sentences", personally, but instead terms. So I don't capitalize them unless they're proper nouns.

Reply:
Fair enough. Add in the `::` then it's good to go.

Reply:
Forgot to push; should be good now!